lib/vscode/src/vs/workbench/api/common/shared/webview.ts

```diff
@@ -51,18 +48,20 @@ export function asWebviewUri(
 		resource = URI.from({
 			scheme: Schemas.vscodeRemote,
 			authority: remoteInfo.authority,
-			path: resource.path,
+			path: resource.path
 		});
 	}
 
-	// NOTE@coder: Add the port separately because if the port is in the domain the
-	// URL will be invalid and the browser will not request it.
+	/**
+	 * @coder Add the port separately because if the port is in the domain the
+	 * URL will be invalid and the browser will not request it.
+	 */
```

Comment:
I think with that new patch upstream, we'll eventually be able to remove this. 

https://github.com/microsoft/vscode/commit/649dd18019332e4dc9090abcf0b98bccf8d57b85